root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4162,41 +4162,48 @@ limited_link_type_combinations: [
                      input_url: 'https://openlibrary.org/authors/OL23919A/',
              input_entity_type: 'artist',
     expected_relationship_type: 'otherdatabases',
+       only_valid_entity_types: ['artist'],
   },
   {
                      input_url: 'https://openlibrary.org/publishers/ISIS_Audio_Books',
              input_entity_type: 'label',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://openlibrary.org/publishers/ISIS_Audio_Books',
+       only_valid_entity_types: ['label'],
   },
   {
                      input_url: 'http://openlibrary.org/publishers/Penguin_Books,_Limited',
              input_entity_type: 'label',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://openlibrary.org/publishers/Penguin_Books,_Limited',
+       only_valid_entity_types: ['label'],
   },
   {
-                     input_url: 'https://www.openlibrary.org/publishers/McGraw-Hill',
+                     input_url: 'https://openlibrary.org/publishers/McGraw-Hill',
              input_entity_type: 'label',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://openlibrary.org/publishers/McGraw-Hill',
+       only_valid_entity_types: ['label'],
   },
   {
                      input_url: 'http://openlibrary.org/books/OL8993487M/Harry_Potter_and_the_Philosopher\'s_Stone',
              input_entity_type: 'release',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://openlibrary.org/books/OL8993487M/',
+       only_valid_entity_types: ['release'],
   },
   {
                      input_url: 'https://openlibrary.org/works/OL20723256W?edition=',
              input_entity_type: 'work',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://openlibrary.org/works/OL20723256W/',
+       only_valid_entity_types: ['work'],
   },
   {
                      input_url: 'https://openlibrary.org/works/OL82592W/',
              input_entity_type: 'work',
     expected_relationship_type: 'otherdatabases',
+       only_valid_entity_types: ['work'],
   },
```

Comment:
This test (not new to this pull request) is redundant with the two previous ones as it has the same `input_url` as their `expected_clean_url` which goes through the same tests. I would suggest to drop it too.